mu4/scenes/inspector/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#=============================================================================
+#  MuseScore
+#  Music Composition & Notation
+#
+#  Copyright (C) 2020 MuseScore BVBA and others
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License version 2.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#=============================================================================
+get_filename_component(MODULE ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+message("Configuring " ${MODULE})
+
+set(MODULE_QRC ${CMAKE_CURRENT_LIST_DIR}/view/inspector_resources.qrc )
+
+set(MODULE_QML_IMPORT ${CMAKE_CURRENT_LIST_DIR}/view/qml)
```

Comment:
I think the directory with qml should be at the root of the module, as it is in other modules.

https://github.com/musescore/Documentation/blob/master/Architecture/Modularity.md

Reply:
Actually, I've missed that somehow

![изображение](https://user-images.githubusercontent.com/57620349/88420393-1adc3500-cde7-11ea-9319-9c768e4d00c6.png)

I think it's might be a confusing thing, because you'd expect to see "qml" in "view", because it's actually the views.

Reply:
See https://github.com/musescore/MuseScore/pull/6372#discussion_r460396830